admin-ui/plugins/auth-server/components/AuthN/LdapForm.js

```diff
@@ -0,0 +1,209 @@
+import { useDispatch } from 'react-redux'
+import { buildLdapPayload } from '../../helper'
+import { STRINGS, ACTIONS, FEATURES } from '../../helper/constants'
+
+function LdapForm({ ldapConfig: initialValues, isEdit, onSuccessApply }) {
+  const [modal, setModal] = useState(false)
+  const formik = useFormik({
+    initialValues,
+    onSubmit: (values, { setSubmitting }) => {
+      setModal(true)
+      setSubmitting(false)
+    },
+  })
+
+  const dispatch = useDispatch()
+  const handleDialogAccept = useCallback(
+    (userMessage) => {
+      const payload = buildLdapPayload(formik.values, userMessage)
+      dispatch({ type: isEdit ? ACTIONS.EDIT_LDAP : ACTIONS.ADD_LDAP, payload, onSuccessApply })
+      setModal(false)
+    },
+    [formik.values, isEdit, dispatch],
+  )
+
+  return (
+    <>
+      <Form onSubmit={formik.handleSubmit} autoComplete="off">
+        <GluuInputRow
+          label={STRINGS.authn.ldap.fields.acr}
+          name="configId"
+          value={formik.values.configId}
+          formik={formik}
+          onChange={formik.handleChange}
+          onBlur={formik.handleBlur}
+          required
+          disabled={isEdit}
+          placeholder={STRINGS.authn.ldap.placeholders.acr}
+          error={formik.touched.configId && formik.errors.configId}
+        />
+        <GluuInputRow
+          label={STRINGS.authn.ldap.fields.level}
+          name="level"
+          type="number"
+          value={formik.values.level}
+          formik={formik}
+          onChange={formik.handleChange}
+          onBlur={formik.handleBlur}
+          required
+          min={1}
+          step={1}
+          placeholder={STRINGS.authn.ldap.placeholders.level}
+          error={formik.touched.level && formik.errors.level}
+        />
+
+        <GluuToogleRow
+          label={STRINGS.authn.ldap.fields.default_authn_method}
+          name="defaultAuthnMethod"
+          formik={formik}
+          value={formik.values.defaultAuthnMethod}
+          handler={() =>
+            formik.setFieldValue('defaultAuthnMethod', !formik.values.defaultAuthnMethod)
+          }
+        />
+        <GluuInputRow
+          label={STRINGS.authn.ldap.fields.bind_dn}
+          name="bindDN"
+          value={formik.values.bindDN}
+          formik={formik}
+          onChange={formik.handleChange}
+          onBlur={formik.handleBlur}
+          required
+          placeholder={STRINGS.authn.ldap.placeholders.bind_dn}
+          error={formik.touched.bindDN && formik.errors.bindDN}
+        />
+        <GluuInputRow
+          label={STRINGS.authn.ldap.fields.max_connections}
+          name="maxConnections"
+          value={formik.values.maxConnections}
+          formik={formik}
+          onChange={formik.handleChange}
+          onBlur={formik.handleBlur}
+          required
+          placeholder={STRINGS.authn.ldap.placeholders.max_connections}
+          error={formik.touched.maxConnections && formik.errors.maxConnections}
+        />
```

Comment:
Add type="number" for maxConnections field

The maxConnections field expects a numeric value but doesn't specify the input type, which could lead to string values being entered.



```diff
         <GluuInputRow
           label={STRINGS.authn.ldap.fields.max_connections}
           name="maxConnections"
+          type="number"
           value={formik.values.maxConnections}
```